README.adoc

```diff
@@ -301,62 +304,67 @@ and trigger a failure if a change introduces a defect.
 
 === Setting up your tests
 
-EndpointTest.java
+EndpointIT.java
 [source, java, linenums, role='code_column hide_tags=comment']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/hateoas/EndpointTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/hateoas/EndpointIT.java[]
 ----
 
 [role="code_command hotspot" subs="quotes"]
 ----
-#Create the `EndpointTest` class.#
-`src/test/java/it/io/openliberty/guides/hateoas/EndpointTest.java`
+#Create the `EndpointIT` class.#
+`src/test/java/it/io/openliberty/guides/hateoas/EndpointIT.java`
 ----
 
-You can use the [hotspot=Before]`@Before` and [hotspot=After]`@After` annotations to perform any setup and teardown tasks for each
+You can use the [hotspot=Before]`@BeforeEach` and [hotspot=After]`@AfterEach` annotations to perform any setup and teardown tasks for each
 of your individual tests.
 
 === Writing the tests
 
-Each test method must be marked with the [hotspot=Test]`@Test` annotation. Typically, the execution order of test
-methods is not controlled, but if control is required, you can place methods in a single container test
-method. This container method is then the only one marked with the [hotspot=Test]`@Test` annotation. The following
-[hotspot=testSuite]`testSuite()` contains two test methods, which run in the order they appear:
-
-The test [hotspot=testLinkForInventoryContents]`testLinkForInventoryContents` is responsible for asserting that
-the correct HATEOAS link is created for the inventory contents.
+Each test method must be marked with the [hotspot=Test1 hotspot=Test2]`@Test` annotation. The execution order of test methods
+is controlled by marking them with the [hotspot=Order1 hotspot=Order2]`@Order` annotation. The value passed into the annotation
+denotes the order in which the methods are run.
```

Comment:
Change
"The value passed into the annotation denotes the order in which the methods are run."
To
"The value that is passed into the annotation denotes the order in which the methods are run."